### PR TITLE
Adding new metrics: pod_cpu_usage_total and pod_memory_working_set

### PR DESCRIPTION
**Pull request by pkkokane28**
# Description of changes
Adding 2 new ContainerInsights metrics which will be used for gaining cost visibility for customers using Enhanced CI: 

- `pod_cpu_usage_total` 
- `pod_memory_working_set`

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests

Deployed the image in test account to validate that the metrics are being emitted.

Screenshot 2024-08-12 at 16 33 45

### CODE REVIEW

**Comment by movence** [comment]
Changes look good to me. Can update the [integ test to validate](https://github.com/aws/amazon-cloudwatch-agent-test/blob/30b4eb1712b48c0858beb44baebb7fd11141fb4b/test/metric_value_benchmark/eks_resources/util.go#L72) these new metrics then attach test result?

**Comment by pkkokane28** [comment]
Screenshot 2024-08-16 at 12 16 27

There is an issue with fetching other metrics in integration Tests, so the complete workflow is not successful: https://github.com/aws/amazon-cloudwatch-agent/actions/runs/10408457860/job/28826102473

**Review by movence — APPROVED** [review]

File: translator/translate/otel/exporter/awsemf/kubernetes.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func getPodMetricDeclarations(conf *confmap.Conf) []*awsemfexporter.MetricDeclar
 		podMetricDeclarations[0].Dimensions = append(podMetricDeclarations[0].Dimensions, []string{"FullPodName", "PodName", "Namespace", "ClusterName"})
 		selectors = append(selectors, []string{"pod_number_of_container_restarts", "pod_number_of_containers", "pod_number_of_running_containers",
 			"pod_status_ready", "pod_status_scheduled", "pod_status_running", "pod_status_pending", "pod_status_failed", "pod_status_unknown",
-			"pod_status_succeeded", "pod_memory_request", "pod_memory_limit", "pod_cpu_limit", "pod_cpu_request",
+			"pod_status_succeeded", "pod_memory_request", "pod_memory_limit", "pod_cpu_limit", "pod_cpu_request", "pod_cpu_usage_total", "pod_memory_working_set",
```
Comment (sky333999) [inline translator/translate/otel/exporter/awsemf/kubernetes.go:102]: nit: keep the memory metric next to the other memory metrics so its easier to find when someone is reading this list.

```
"pod_memory_request", "pod_memory_limit", "pod_memory_working_set", "pod_cpu_limit", "pod_cpu_request", "pod_cpu_usage_total",
```

**Review by sky333999 — APPROVED** [review]

File: translator/translate/otel/exporter/awsemf/kubernetes.go (hunk 1 above)
Comment (pkkokane28) [reply translator/translate/otel/exporter/awsemf/kubernetes.go:102]: sure, will fix it in next PR

**Pull request merged**